src/axios/apiQueries.js

```diff
@@ -21,3 +21,16 @@ export const getCommentForSingleReview = (review_id) => {
     return commentsResponse.data.comments;
   });
 };
+
+export const getUpdateVotes = (review_id, inc_votes) => {
+  return api
+    .patch(`/reviews/${review_id}`, {
+      inc_votes,
+    })
+    .then((voteResponse) => {
+      return voteResponse;
+    })
+    .catch((error) => {
```

Comment:
You shouldn't put `.catch` blocks inside these util functions. This will catch any errors and swallow them so the promise will resolve with `undefined` when you use it in the component. 

As a general rule you should put put blocks on the end of your promise chains, in the component in this case. If this promise rejects and is unhandled you'll still see the error in your console so you don't need to log it yourself FYI